src/emonesp.h

```diff
@@ -5,15 +5,7 @@
 // General support code used by all modules
 // -------------------------------------------------------------------
 
-// Uncomment to use hardware UART 1 for debug else use UART 0
-// #define DEBUG_SERIAL1
```

Comment:
I think the default should be Serial1 for debug as Serial is connected to OpenEVSE. However I do use Serial for debug sometime when just developing without a connected OpenEVSE. Was probably a mistake to commit anything else. I have also been transitioning to the macros in debug.h as these go away in production builds (I think I will rename openevse-ota to openevse-dev)